protobuf/src/main/proto/message_example_2.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+package io.apicurio.schema.validation.protobuf.ref;
+
+message MessageExample2 {
+
+  string key2 = 1;
+  int32 value2 = 2;
+
```

Comment:
I see a lot of logic around handling references, should we have a test for them?

Reply:
Actually, the schema parser has been extracted from our serdes. References are not supported yet in this library. They will, but not yet, so no tests needed for now.